Cargo.toml

```diff
@@ -22,15 +22,15 @@ rust_team_data = { git = "https://github.com/rust-lang/team" }
 glob = "0.3.0"
 toml = "0.5.1"
 hyper = "0.13"
```

Comment:
I'd also update `hyper` and `octocrab` to the latest so that we can drop tokio 0.2.x completely.

Reply:
@JohnTitor the reason I was suggesting to hold back updating these two crates is that they have both have a conflict in a common nested dependency (the `httparse` crate). [There's a PR](https://github.com/dekellum/hyperx/pull/32) that could potentially fix this for us, so I'd like to wait and see.

Otherwise I'm not sure how can I pin the right version for `httparse` so that `octocrab` and `hyper` can get along happily (any suggestion?)

Reply:
for context
```
error: failed to select a version for `httparse`.
    ... required by package `hyperx v1.3.0`
    ... which is depended on by `octocrab v0.9.1`
    ... which is depended on by `triagebot v0.1.0 (../rust/triagebot)`
versions that meet the requirements `>=1.0, <1.4` are: 1.3.6, 1.3.5, 1.3.4, 1.3.3, 1.3.2, 1.3.1, 1.3.0, 1.2.5, 1.2.4, 1.2.3, 1.2.2, 1.2.1, 1.2.0, 1.1.2, 1.1.1, 1.1.0, 1.0.0

all possible versions conflict with previously selected packages.

  previously selected package `httparse v1.4.0`
    ... which is depended on by `hyper v0.14.9`
    ... which is depended on by `hyper-tls v0.5.0`
    ... which is depended on by `reqwest v0.11.4`
    ... which is depended on by `octocrab v0.9.1`
    ... which is depended on by `triagebot v0.1.0 (../rust/triagebot)`

failed to select a version for `httparse` which could resolve this conflict
```

Reply:
uhm ... maybe I can work around the issue by carefully pinning `hyper = "0.14.5" ` (... checking...)

Reply:
We can avoid it by using hyper 0.14.5 or lower, I mean, we don't have to use the latest patch version to drop tokio 0.2, a newer minor version is enough. You can specify it as follows:

```toml
hyper = { version = "=0.14.5", features = ["server"] } # Note: "server" feature is required on 0.14.x.
```

I haven't tested deeply but it should resolve the dependency issue.